### PR TITLE
MDO Detection Details Report PowerBI template

### PR DESCRIPTION
**Pull request by Justfyn**
Required items, please complete
   
   Change(s):
   -Added folder under 'Azure-Sentinel/Tools' called 'MDO Power BI Dashboard'
   -Uploaded 'MDO Detection Details Report v2.2.pbit' PowerBI file
   -Added Readme.md files with details how to configure the PowerBI file
   -Added folder under 'Azure-Sentinel/Tools/MDO Power BI Dashboard' called 'Media' to store images for markdown file

   Reason for Change(s):
   - Existing project for MDO dashboard 

   Version Updated:
   - No

   Testing Completed:
   - Yes, discussed over email

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-spadarthi** [comment]
@v-rucdu : Please have a look and provide an update.Thanks!

File: Tools/MDO Power BI Dashboard/Readme.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+[Overview](#overview)<br/>
+[Example View of Microsoft Defender for Office 365 (MDO) Detection Details Report](#example-view-of-microsoft-defender-for-office-365-mdo-detection-details-report)<br/>
+[How to use the .pbit file](#how-to-use-the-pbit-file)<br/>
+[How to publish to Power BI online and configure scheduled auto-refresh](#how-to-publish-to-power-bi-online-and-configure-scheduled-auto-refresh)<br/>
+
+## Overview
+
+This report template will give you an example how to use Microsoft 365 Defender Hunting API to built a custom report using Power BI. This way you can visualise Microsoft Defender for Office 365 (MDO) data based on your organisation needs. 
+
+* It is using Delegated model to connect to the M365D Hunting API. No need for app regirstartion simply need an admin account which can run the underlying Hunting queries using the API.
```
Comment (v-rucdu) [inline Tools/MDO Power BI Dashboard/Readme.md:14]: typo: registration

File: Tools/MDO Power BI Dashboard/Readme.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+[How to use the .pbit file](#how-to-use-the-pbit-file)<br/>
+[How to publish to Power BI online and configure scheduled auto-refresh](#how-to-publish-to-power-bi-online-and-configure-scheduled-auto-refresh)<br/>
+
+## Overview
+
+This report template will give you an example how to use Microsoft 365 Defender Hunting API to built a custom report using Power BI. This way you can visualise Microsoft Defender for Office 365 (MDO) data based on your organisation needs. 
+
+* It is using Delegated model to connect to the M365D Hunting API. No need for app regirstartion simply need an admin account which can run the underlying Hunting queries using the API. 
+* The built-in Security Reader or Security Administrator role is enough to have the report working for example. 
+* This is inteneded to be a template, we encourage everybody to modify queries, visualisations, bring in more data sets based on organisation needs.
```
Comment (v-rucdu) [inline Tools/MDO Power BI Dashboard/Readme.md:16]: typo: intended

File: Tools/MDO Power BI Dashboard/Readme.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Overview
+
+This report template will give you an example how to use Microsoft 365 Defender Hunting API to built a custom report using Power BI. This way you can visualise Microsoft Defender for Office 365 (MDO) data based on your organisation needs. 
+
+* It is using Delegated model to connect to the M365D Hunting API. No need for app regirstartion simply need an admin account which can run the underlying Hunting queries using the API. 
+* The built-in Security Reader or Security Administrator role is enough to have the report working for example. 
+* This is inteneded to be a template, we encourage everybody to modify queries, visualisations, bring in more data sets based on organisation needs. 
+* The “ReadMe” tab of the template files has more information about terminology used in the template.  
+* This not intended to be a permanent or complete solution rather show an example how to create custom Microsoft Defender for Office 365 (MDO) reports unsing the hunting API and Power BI.
```
Comment (v-rucdu) [inline Tools/MDO Power BI Dashboard/Readme.md:18]: typo: using

File: Tools/MDO Power BI Dashboard/Readme.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+## Contents
+
+[Overview](#overview)<br/>
+[Example View of Microsoft Defender for Office 365 (MDO) Detection Details Report](#example-view-of-microsoft-defender-for-office-365-mdo-detection-details-report)<br/>
+[How to use the .pbit file](#how-to-use-the-pbit-file)<br/>
+[How to publish to Power BI online and configure scheduled auto-refresh](#how-to-publish-to-power-bi-online-and-configure-scheduled-auto-refresh)<br/>
+
+## Overview
+
+This report template will give you an example how to use Microsoft 365 Defender Hunting API to built a custom report using Power BI. This way you can visualise Microsoft Defender for Office 365 (MDO) data based on your organisation needs.
```
Comment (v-rucdu) [inline Tools/MDO Power BI Dashboard/Readme.md:12]: typo: use build instead of built

**Review by v-rucdu — CHANGES_REQUESTED** [review]
@Justfyn: there are few typos in the read me file, please fix them and we are good to merge post that... Thanks!!!

**Review by v-rucdu — APPROVED** [review]

**Pull request merged**